packages/firebase_data_connect/analysis_options.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
Same thing for this file

Reply:
Even if we want to ignore the generated files in the linter?